torch/nn/modules/linear.py

```diff
@@ -44,9 +44,9 @@ class Linear(Module):
             Default: ``True``
 
     Shape:
-        - Input: :math:`(N, *, H_{in})` where :math:`*` means any number of
-          additional dimensions and :math:`H_{in} = \text{in\_features}`
-        - Output: :math:`(N, *, H_{out})` where all but the last dimension
+        - Input: :math:`(*, H_{in})` where :math:`*` means any number of
+          additional dimensions including none and :math:`H_{in} = \text{in\_features}`.
```

Comment:
I know this is pretty pedantic, but I think "additional" isn't quite right here now that `*` comes first